src/set_theory/ordinal_arithmetic.lean

```diff
@@ -1853,6 +1859,9 @@ theorem omega_le {o : ordinal.{u}} : omega ≤ o ↔ ∀ n : ℕ, (n : ordinal)
    let ⟨n, e⟩ := lt_omega.1 h in
    by rw [e, ← succ_le]; exact H (n+1)⟩
 
+theorem omega_eq_sup_nat : omega.{u} = sup (λ n : ℕ, n) :=
```

Comment:
This will make it clearer what's going on
```suggestion
theorem omega_eq_sup_nat_cast : omega.{u} = sup nat.cast :=
```

Reply:
That's really clever!